tests/console/procps.pm

```diff
@@ -30,6 +30,7 @@ use base 'opensusebasetest';
 use testapi;
 use utils;
 use strict;
+use Time::HiRes 'sleep';
```

Comment:
This is not needed.

Reply:
Removed and submitted the latest changes